src/store/reducers/shardsWorkload.ts

```diff
@@ -1,22 +1,30 @@
+import {Reducer} from 'redux';
+
 import '../../services/api';
+import type {IShardsWorkloadAction, IShardsWorkloadState} from '../../types/store/shardsWorkload';
 
 import {parseQueryAPIExecuteResponse} from '../../utils/query';
 
 import {createRequestActionTypes, createApiRequest} from '../utils';
 
-const SEND_SHARD_QUERY = createRequestActionTypes('query', 'SEND_SHARD_QUERY');
+export const SEND_SHARD_QUERY = createRequestActionTypes('query', 'SEND_SHARD_QUERY');
 const SET_SHARD_QUERY_OPTIONS = 'query/SET_SHARD_QUERY_OPTIONS';
 
 const initialState = {
     loading: false,
     wasLoaded: false,
 };
 
-function formatSortOrder({columnId, order}) {
+export interface SortOrder {
+    columnId: string;
+    order: string;
+}
+
+function formatSortOrder({columnId, order}: SortOrder) {
     return `${columnId} ${order}`;
 }
 
-function createShardQuery(path, sortOrder, tenantName) {
+function createShardQuery(path: string, sortOrder?: SortOrder[], tenantName?: string) {
     const orderBy = Array.isArray(sortOrder) ?
```

Comment:
`sortOrder` is of Array type as function requires it. So we can check just `sortOrder ? ... : ""`